dash/development/_py_components_generation.py

```diff
@@ -22,13 +22,21 @@
 import_string = """# AUTO GENERATED FILE - DO NOT EDIT
 
 import typing  # noqa: F401
-import numbers # noqa: F401
 from typing_extensions import TypedDict, NotRequired, Literal # noqa: F401
-from dash.development.base_component import Component
-try:
-    from dash.development.base_component import ComponentType # noqa: F401
-except ImportError:
```

Comment:
Note: types defined in try except don't always resolve.

Reply:
that makes sense now that you've said it but damn - I'm liking Python's type system less and less as time goes by…